### DISCUSSION TITLE
ValueError: too many values to unpack (expected 6)

### DISCUSSION THREAD

**Discussion by garymmi** [snakers4/silero-vad#57] [Q&A] [answered]
`after` update the code, it shows

Traceback (most recent call last):
  File "vad2.py", line 43, in <module>
    _, single_audio_stream, _) = utils
ValueError: too many values to unpack (expected 6)

usage:
```
model, utils = torch.hub.load(repo_or_dir='snakers4/silero-vad',
                              model='silero_vad',
                              force_reload=True)
(get_speech_ts,
 _, read_audio,
_, single_audio_stream, _) = utils
```

**Comment by snakers4** [comment, marked as answer]
https://github.com/snakers4/silero-vad/issues/56

**Reply by snakers4** [reply]
Also solved
